ci/conan/all/conandata.yml

```diff
@@ -45,21 +69,21 @@ sources:
 patches:
   "8.0.1":
     - patch_file: "patches/8.0.0-0005-install-utils.patch"
-      patch_description: "enable utils installation"
+      patch_description: "enable utilis installation"
```

Comment:
is this a typo? It appears three times

Reply:
I copied these lines from upstream... Reverted

Reply:
Oh, could you upstream this too?

Reply:
I'll wait until https://github.com/conan-io/conan-center-index/pull/21669 gets checked in. But I'm not sure if it can get reviewed quickly.